tests/accelerators/test_dp.py

```diff
@@ -95,3 +98,54 @@ def test_dp_test(tmpdir):
     new_weights = model.c_d1.weight.clone().detach().cpu()
 
     assert torch.all(torch.eq(old_weights, new_weights))
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
```

Comment:
We won't be really training for this tests right? Can you mock the device_count?

Reply:
It needs the GPUs to initialize the DP plugin else I think it stays on CPU.

Reply:
Seems to work: https://github.com/PyTorchLightning/pytorch-lightning/pull/6098/commits/0034c5c6e129f274cf0dbb35ba4e455db56eac67

Reply:
oh okay.. thanks :v: